test/ffmpeg-vaapi/encode/10bit/vp9.py

```diff
@@ -16,7 +16,7 @@ class VP9_10EncoderBaseTest(EncoderTest):
   def before(self):
     super().before()
     vars(self).update(
-      codec    = "vp9",
+      codec    = "vp9-10",
```

Comment:
The lib/ffmpeg/vaapi/encoder.py has some conditional checks for self.codec "vp9".  Need to update them to include "vp9-10" with this change.

Also, mapprofile in ffmpeg/vaapi/util.py...

Reply:
updated, thanks